tests/Unit/DataStructures/Tensor/Expressions/Test_AddSubSymmetry.cpp

```diff
@@ -4,17 +4,51 @@
 #include "Framework/TestingFramework.hpp"
 
 #include <cstddef>
+#include <string>
 #include <type_traits>
 
 #include "DataStructures/Tensor/Symmetry.hpp"
 #include "DataStructures/Tensor/Tensor.hpp"
+#include "Utilities/ErrorHandling/Error.hpp"
 #include "Utilities/TMPL.hpp"
 
 namespace {
 template <auto&... TensorIndices>
 using make_tensorindex_list =
     tmpl::list<std::decay_t<decltype(TensorIndices)>...>;
 
+void test_impl_consistency() {
+  const std::string error_msg =
+      ("Symmetry and AddSubSymmetry are no longer using the same canonical "
```

Comment:
just an FYI, unlike python C++ doesn't care about white space so you strictly don't need the parens here

Reply:
Thanks, I'll go ahead and remove them when I squash the current fixups!